Sources/SwiftKuery/QuerySuffixProtocol.swift

```diff
@@ -0,0 +1,23 @@
+/**
+ Copyright IBM Corporation 2016, 2017
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+// MARK: QuerySuffixProtocol
+
+/// Defines the protocol which should be used for all suffix clauses.
+/// Represents a suffix as String value.
```

Comment:
The method build() represents suffixes as Strings, not the protocol itself. Could you please remove the last line of the comment.